api/pom.xml

```diff
@@ -4,7 +4,7 @@
 
```

Comment:
No no no no no. Why this is in 3rd PR? Please remove this. Fix PR where I told you to leave it and then update this PR